src/Metrics.tcl

```diff
@@ -254,5 +254,253 @@ proc report_design_area_metrics {args} {
   utl::metric_float "design__instance__utilization__stdcell" $stdcell_util
 }
 
+proc report_puts { out } {
+    upvar 1 when when
+    upvar 1 filename filename
+    set fileId [open $filename a]
+    puts $fileId $out
+    close $fileId
+}
+
+define_cmd_args "report_metrics" {[-stage][-when][-include_erc][-include_clock_skew][-metrics_report_dir]}
+proc report_metrics { stage when {include_erc true} {include_clock_skew true} {metrics_report_dir ""}} {
+  if {[info exists ::env(SKIP_REPORT_METRICS)] && $::env(SKIP_REPORT_METRICS) == 1} {
+    return
+  }
+  puts "Report metrics stage $stage, $when..."
+  if {$metrics_report_dir eq ""} {
+    set metrics_report_dir [file join  $::env(HOME) "OpenROAD" "results"]
+  }
```

Comment:
Creating directories in the user's home is unexpected. I think '.' is a more reasonable default.

Reply:
If a user has only read access in the '.' directory, it will not work, but the user usually has write access to his home directory.

Reply:
If they don't they can use -metrics_report_dir.  I would not be happy to use a tool that behaved like this.

Reply:
Well. I updated the code, based on your suggestion.